src/Struct/PullRequest.php

```diff
@@ -37,11 +37,6 @@ abstract class PullRequest
      */
     public array $rawCommits = [];
 
-    /**
-     * @var array{'changes': array{'new_path': string, 'diff'?: string, 'new_file': bool, 'deleted_file': bool}[]}
-     */
-    public array $rawFiles = ['changes' => []];
```

Comment:
phpstan complained now (don't know why not before) because in the github pr class, we don't use the `changes` key, the formats are different. So I removed from base class and typed separately in the implementations.